tests/_cli/snapshots/ipynb.txt

```diff
@@ -3,35 +3,35 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0e29fba8",
+   "id": "cd2168d6",
```

Comment:
Non-deterministic so will fail again

I think in cell creation you can pass in "id" as a kwarg- could just use the marimo cell_id, I think it's loose